mopidy_local/www/index.html

```diff
@@ -1,43 +1,44 @@
 <!doctype html>
 <html>
   <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Mopidy-Local</title>
+    <link rel="stylesheet" href="/mopidy/mopidy.css" type="text/css" media="screen, print">
     <style>
-      html {
-          background: #f8f8f8;
-          color: #555;
-          font-family: Geneva, Tahoma, Verdana, sans-serif;
-          line-height: 1.4em;
+      #thumbnail_container {
+       display: grid;
+       grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+       grid-gap: 0px;
+       line-height: 0%;
       }
-      body {
-          margin: 0 auto;
-          max-width: 600px;
-      }
-      h1 {
-          font-weight: 500;
-          line-height: 1.1em;
-      }
-      a {
-          color: #555;
-          text-decoration: none;
-      }
-      img {
-          border: 0;
-          height: 80px;
-          width: 80px;
+      .thumbnail {
+          border: 0px none;
+          max-width: 98%;
+          max-height: 96%;
+          height: auto;
+          width: auto;
       }
     </style>
   </head>
   <body>
-    <h1>Mopidy-Local</h1>
-    <p>
-      This Web client is used to serve album art extracted from local
-      media files by the Mopidy-Local extension.
-    </p>
-    <div>
-      {% for image in images %}
-        <a href="{{image}}"><img src="{{image}}"></a>
-      {% end %}
+    <div class="box focus">
+      <h1>Mopidy-Local</h1>
+      <p>
+        This Web client is used to serve album art extracted from local
+        media files by the Mopidy-Local extension.
+      </p>
+    </div>
+    <div class="box">
+      <h2 id="b1">Mopidy-Local image cache</h2>
+      <span id="thumbnail_container">
+        {% for image in images %}
+        <a style="border: 0px none;" href="{{image}}">
+          <img class="thumbnail" src="{{image}}" loading="lazy">
```

Comment:
`<img>` tags aren't valid unless they have an `alt` attribute. Could we add one here, even if it's just something simple like "Album Thumbnail"?

Reply:
Yes, can do. I think "Album art thumbnail" makes sense. It would be nice to be able to say something more descriptive (and I will be able to do that in the `mopidy-youtube` version, I think) but for `mopidy-local`,  `web.py` just traverses the cache folder and picks up filenames; I don't think there's any point giving the mopidy-local filename, which is just an md5 hash + image size, I think.

Reply:
It looks like tornado templates are a very thing abstraction over python code. Perhaps you could do something like this?

```
{% for idx, image in enumerate(images) %}
<img src="{{ image }}" alt="Thumbnail {{ idx + 1 }}"
{% endfor %}
```

That's just gold-plating though. It's more about personal curiosity for learning how Tornado templates work than anything else.

Reply:
Yeah, very thin. I had a play around with the template a bit over on `mopidy-youtube`. Getting the track title as alt text was pretty easy in the end, primarily because in `mopidy-youtube` the metadata is cached along with the image. 

But I think here, adding an index number doesn't really tell you anything more about the image or the track that it is associated with. So I'm leaning towards leaving it out. Presumably, with a bit of code, the metadata could be retrieved from the database, but that's further than I'm prepared to go for this pretty niche 'enhancement'. How many people have ever looked at `http://127.0.0.1:6680/local`, anyway? Lol.